javascript/javascript1/week2/exerciseStep3Week2.js

```diff
@@ -0,0 +1,138 @@
+//Flight Booking Full Name Function//////////////////////////////////////////////////////////////
+
+function getFullName(firstname, surname, useFormalName) {
+    if (firstname === '') {
+        return 'Please write your first name'
+    } else if (surname === '') {
+        return 'Please write your surname'
+    } else if (useFormalName) {
+        return 'Lord ' + firstname + ' ' + surname;
+    } else {
+        return firstname + ' ' + surname;
+    }
+}
+
+const fullName1 = getFullName('Hamed', 'Karachi', true);
+const fullName2 = getFullName('Arman', 'Naderi', false);
+
+console.log(fullName1);
+console.log(fullName2);
+
+//I do not understand the last section of this exercise when it said we should try out to put it in codepen
+
+
+//Event Application/////////////////////////////////////////////////////////////////////////
+//"i" is the parameter for how many day we have until to the event
+
+const n = new Date().getDay();
+const weekday = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+
+function getEventWeekday(i) {
+    if (i >= 7) {
+        return ((i % 7) + n) - 1;
+    } else {
+        return ((i + n) % 7) - 1;
+    }
+}
```

Comment:
I think this function may be missing a few things. Seems like it's returning a number and not a weekday. Also, test with giving it an argument that is larger than 7

Reply:
Oh , yes. Of course. I have added changes now.

Reply:
I do not know if you can see changes or not!!I get confused as I add and commit changes and push it again in this place!!!!